diff --git a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
index 4ec4514..10e2849 100644
--- a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
+++ b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
@@ -91,8 +91,11 @@ ext.useExpoPublishing = {
   project.afterEvaluate {
     publishing {
       publications {
-        release(MavenPublication) {
-          from components.release
+        def releaseComponent = components.findByName("release")
+        if (releaseComponent != null) {
+          release(MavenPublication) {
+            from releaseComponent
+          }
         }
       }
       repositories {
